src/client/src/apps/MainRoute/widgets/spotTile/components/Tile/Tile.test.tsx

```diff
@@ -30,7 +30,7 @@ const defaultParams: Parameters<typeof getDerivedStateFromUserInput>[0] = {
       mid: 1.48357,
       priceMovementType: 'Down' as PriceMovementTypes,
       symbol: 'EURCAD',
-      valueDate: '2019-04-07T00:00:00Z',
+      valueDate: new Date(2019, 3, 7).toISOString(),
```

Comment:
These look like more changes that aren't part of your work, but for which the commits have somehow been pulled in? The one below as well?

Reply:
Yup seems to be an issue with the rebase I did. I will just create a new branch from my master and copy over my changes to the FooterVersion.tsx file.